portfolio/src/main/webapp/script.js

```diff
@@ -12,37 +12,42 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-var curSlide = 0; //by default, display first slide
-const totalSlides = 3; //total number of slides
+let curSlide = 0; //By default, display first slide.
 
 /*
- * Advance forward by one slide
+ * Advance forward by one slide.
  */
 function forward() {
-  if (curSlide < totalSlides - 1) {
-    updateSlide(curSlide + 1);
-  } else { //curSlide = totalSlides
-    updateSlide(0);
-  }
+  updateSlide(curSlide + 1);
 }
 
 /*
- * Advance backwards by one slide
+ * Advance backwards by one slide.
  */
 function back() {
-  if (curSlide == 0) {
-    updateSlide(totalSlides - 1);
-  } else {
-    updateSlide(curSlide - 1);
-  }
+  updateSlide(curSlide - 1);
```

Comment:
Nice. I'm a fan of how the complexity has been contained to `updateSlide` so the callers don't have to deal with it.